library/Kubernetes/Web/ClusterForm.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/* Icinga for Kubernetes Web | (c) 2024 Icinga GmbH | AGPLv3 */
+
+namespace Icinga\Module\Kubernetes\Web;
+
+use Generator;
+use Icinga\Module\Kubernetes\Common\Database;
+use Icinga\Module\Kubernetes\Model\Cluster;
+use ipl\Web\Compat\CompatForm;
+use Ramsey\Uuid\Uuid;
+
+class ClusterForm extends CompatForm
+{
+    public const ALL_CLUSTERS = '-';
+
+    protected function yieldClusters(): Generator
+    {
+        $clusters = Cluster::on(Database::connection())
+            ->columns(['uuid', 'name']);
+
+        foreach ($clusters as $cluster) {
+            yield (string) Uuid::fromBytes($cluster->uuid) => $cluster->name;
```

Comment:
You could do `$cluster->name ?? $cluster->uuid` here and don't do it in `assemble()`.